Showcase/Application/Foundations/Assets/AssetsViewController.swift

```diff
@@ -461,5 +461,6 @@ extension AssetsViewController {
                           image: VitaminAssets.Placeholder.landscape.image)
             ])
         ]
+        return tempSections
```

Comment:
I forgit to do a rollback : I created the variable because I wanted to use it as a fonction parameter I called just before, but found another way (with the didSet on the `sections variable)`, and forgot to remove it.